lib/ood_core/job/adapters/torque/batch.rb

```diff
@@ -440,15 +447,17 @@ def qsub_submit(script, queue, headers, resources, envvars)
           "PBS_DEFAULT"     => "#{host}",
           "LD_LIBRARY_PATH" => "#{lib}:#{ENV['LD_LIBRARY_PATH']}"
         }
-        cmd = bin.join("qsub").to_s
+        cmd = OodCore::Job::Adapters::Helper.bin_path('qsub', bindir, custom_bin)
```

Comment:
Did you mean `bin` instead of `bindir` here?

Reply:
Yes, yes I did mean that.